template_debian/distribution.sh

```diff
@@ -476,9 +476,22 @@ EOF
 # Uninstall Qubes Repo
 # ==============================================================================
 uninstallQubesRepo() {
-    info ' Removing Qubes build repo from sources.list.d'
+    info "Removing Qubes build repo from sources.list.d"
 
     # Lets not umount; we do that anyway when 04 exits
     umount_kill "${INSTALL_DIR}/tmp/qubes_repo"
     rm -f "${INSTALL_DIR}/etc/apt/sources.list.d/qubes-builder.list"
 }
+
+
+# ==============================================================================
+# Install Mozilla/PPA Repo
+# ==============================================================================
+installMozillaRepo() {
+    cat > "${INSTALL_DIR}/etc/apt/sources.list.d/qubes-builder.list" <<EOF
+deb https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu ${DIST_CODENAME} main
```

Comment:
This should use `[signed-by=...]` instead of adding the key to the globally trusted set. Especially with the remark below...